developer-docs-site/docs/tutorials/index.md

```diff
@@ -22,6 +24,22 @@ If running macOS, install the following packages in the order specified to take
     ```
 1. **Poetry**: Install Poetry from [https://python-poetry.org/docs/#installation](https://python-poetry.org/docs/#installation).
 
+## Adjust network and funding source
```

Comment:
I don't think we should recommend they use testnet with the tutorials. Perhaps we could clean this up?

Reply:
Removed and rewritten.